group_vars/all.yml

```diff
@@ -43,6 +43,25 @@ adw:
 sync:
   version: 3.6.0
 
+supported_os:
+  RHEL:
+    versions:
+      - 8.4
+      - 8.2
+      - 7.7
+      - 7.6
+  CentOS:
+    versions:
+      - 7
```

Comment:
set it to 7.9 (which is latest CEntos anything older than that is anyway a bad idea ... w7.9. itself is no good idea :) )